### DISCUSSION TITLE
After update I got error

### DISCUSSION THREAD

**Discussion by moresol** [rose-pine/neovim#220] [Q&A] [answered]
How to fix?
![Screenshot 2024-01-27 at 12 52 18 AM](https://github.com/rose-pine/neovim/assets/106124331/108c6070-6e9b-4159-9c91-042ca8da758a)
![Screenshot 2024-01-27 at 12 52 07 AM](https://github.com/rose-pine/neovim/assets/106124331/30329eb8-d0df-4008-b81a-542b75f34579)

**Comment by mvllow** [comment]
Looking into this.

Edit: Looks like this is an error with headings being a table _and_ the `Search` highlight group override, presumably because of `inherit`. The headings were changed to be flat, e.g. instead of `headings.h1` it's just `h1` but this should not have been breaking. The inherit key should not cause issues either, so these are both bugs and I apologise for the interruption. Will push out an update asap.

**Reply by mvllow** [reply, marked as answer]
Pushed an update to address both issues—hopefully :)

Thank you for reporting this, very much appreciate it
